src/charset.c

```diff
@@ -849,16 +849,87 @@ linetabsize_no_outer(win_T *wp, linenr_T lnum)
 #endif
 }
 
+static int s_use_vcol_cache = FALSE;
+static int s_vcol_cache_valid1 = FALSE;
+static int s_vcol_cache_valid2 = FALSE;
+
+    void
+enable_vcol_cache(void)
+{
+    s_use_vcol_cache = TRUE;
+}
+
+    void
+disable_vcol_cache(void)
+{
+    s_use_vcol_cache = TRUE;
```

Comment:
Maybe it's like this?

```suggestion
    s_use_vcol_cache = FALSE;
```

Reply:
thanks